mail/templates/product_order_receipt/body.html

```diff
@@ -0,0 +1,88 @@
+{% extends "email_base.html" %}
+
+{% block content %}
+<tr class="receipt-new-wrapper" style="font: 14px/22px 'Arial', sans-serif; margin: 0 auto; padding: 0 20px">
+    <table style="margin: 0 20px !important;">
+    <td class="rec-head" style="color: rgba(0,0,0,0.85); font: 14px/20px 'Arial', sans-serif; max-width: 696px; ">
+        <p style="font-weight: normal; margin: 0 0 20px;">Dear {{ purchaser.name }},</p>
+        <p style="font-weight: normal; margin: 0 0 20px;">You have been enrolled {% if content_title %} in {{ content_title }}{% endif %}.
```

Comment:
Can we include the course number here as well?

e.g. "14.740x Foundations of Development Policy"